src/common/config.h

```diff
@@ -192,6 +192,9 @@ struct md_config_t {
   /// get encoded map<string,string> of compiled-in defaults
   void get_defaults_bl(const ConfigValues& values, ceph::buffer::list *bl);
 
+  // Get the default value of a configuration option
+  int get_val_default(const std::string_view key, std::string *val);
```

Comment:
Delete the 'const'.

Also: I'd suggest using a different API (as we should for most such cases):
std::optional<std::string> get_val_default(std::string_view key);

Reply:
1. Ack on deleting 'const'.

2. I will rewrite the get_val_default() method according to your suggestion.

Reply:
@ronen-fr Done. Please verify.